### DISCUSSION TITLE
When I delete the comics in lanraragi, the files in the hard disk will also be deleted directly.

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#714] [Q&A] [answered]
I was using the Windows version of lanraragi. Whenever I delete a manga in lanraragi, the files in the hard disk are also completely deleted, and they don’t even enter the recycle bin! Is there a way to just delete the manga displayed in lanraragi instead of directly deleting the files on the hard disk?

**Comment by Difegue** [comment, marked as answer]
I'd suggest making your content folder [read-only](https://superuser.com/questions/868399/how-to-make-a-folder-read-only-in-windows) in case you don't want LRR to delete files. (Which only happens when you delete archives, as the software will tell you)  
I think WSL1 will send deleted files to the recycle bin since that uses normal Windows kernel APIs, but WSL2 will certainly just delete stuff outright.
